stable/syncthing/2.1.0/Chart.yaml

```
@@ -0,0 +1,32 @@
+apiVersion: v2
+kubeVersion: ">=1.16.0-0"
+name: syncthing
+version: 2.1.0
+upstream_version: 1.15
+appVersion: "auto"
+description: P2P file synchronization application
+type: application
+deprecated: false
+home: https://syncthing.net/
+icon: https://raw.githubusercontent.com/syncthing/syncthing/main/assets/logo-128.png
+keywords:
+  - syncthing
+sources:
+  - https://github.com/syncthing/syncthing
```

In relation to changing above home url:
```suggestion
sources:
  - https://syncthing.net/
  - https://github.com/syncthing/syncthing
```